src/main/java/nextstep/subway/config/WebMvcConfig.java

```diff
@@ -1,23 +1,40 @@
 package nextstep.subway.config;
 
+import org.springframework.boot.web.servlet.FilterRegistrationBean;
+import org.springframework.context.annotation.Bean;
 import org.springframework.context.annotation.Configuration;
 import org.springframework.http.CacheControl;
+import org.springframework.web.filter.ShallowEtagHeaderFilter;
 import org.springframework.web.servlet.config.annotation.ResourceHandlerRegistry;
 import org.springframework.web.servlet.config.annotation.WebMvcConfigurer;
 
+import javax.servlet.Filter;
+
 @Configuration
 public class WebMvcConfig implements WebMvcConfigurer {
     public static final String PREFIX_STATIC_RESOURCES = "/resources";
 
     @Override
     public void addResourceHandlers(ResourceHandlerRegistry registry) {
-        registry.addResourceHandler(PREFIX_STATIC_RESOURCES + "/static/**")
-                .addResourceLocations("classpath:/static/")
-                .setCacheControl(CacheControl.noCache().cachePrivate());
-
         registry.addResourceHandler(PREFIX_STATIC_RESOURCES + "/static/js/**")
                 .addResourceLocations("classpath:/static/js/")
                 .setCacheControl(CacheControl.noCache().cachePrivate());
 
+        registry.addResourceHandler(PREFIX_STATIC_RESOURCES + "/static/images/**")
```

Comment:
images 로 해주신 이유가 있으신가요? 😄 
```suggestion
        registry.addResourceHandler(PREFIX_STATIC_RESOURCES + "/static/**")
```

Reply:
아닙니다!! images만 따로 해보고 전체만 해봤던 흔적이네요 ㅜㅜ